tests/schema/test.ridl

```diff
@@ -1,9 +1,24 @@
-webrpc = v1             # version of webrpc schema format (ridl or json)
+webrpc = v1
 
-name = Test             # name of your backend app
-version = v0.0.1        # version of your schema
+name = Test
+version = v0.10.0
 
-struct Empty
+service TestApi
+  - GetEmpty()
+  - GetError()
+  
+  - GetOne() => (one: Simple)
+  - SendOne(one: Simple)
+
+  - GetMulti() => (one: Simple, two: Simple, three: Simple)
+  - SendMulti(one: Simple, two: Simple, three: Simple)
+  
+  - GetComplex() => (complex: Complex)
+  - SendComplex(complex: Complex)
```

Comment:
This is the updated Interoperability TestApi schema. All generators should run tests against reference webrpc-test server.